.cloudbuild/release.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+set -o errexit -o xtrace
+
+# setup remote cache
+curl -fsSL https://github.com/buchgr/bazel-remote/releases/download/v2.4.0/bazel-remote-2.4.0-linux-x86_64 -o bazel-remote
```

Comment:
can you ensure the hash here?
comparing sha256sum bazel-remote to some fixed value?